spec/integration/inventory_spec.rb

```diff
@@ -110,36 +131,36 @@ module Hawkular::Inventory::RSpec
     end
 
     it 'Should list children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       expect(resources.size).to be(1)
 
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly)
 
-      expect(children.size).to be(21)
+      expect(children.size).to be(22)
     end
 
     it 'Should list recursive children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly, recursive: true)
 
-      expect(children.size).to be(211)
+      expect(children.size).to be(251)
```

Comment:
I didn't explain myself well. I thought that this PR's goal is to add support for erb templating as a general infra feature, so I was wondering why there was a need to update numbers in the specific tests.